.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.1
+jobs:
+  build_and_test:
+    executor: node/default
+    steps:
+      - checkout
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          name: Run tests
+          command: npm test
+      - run:
+          name: Run build
+          command: npm run build
+      - persist_to_workspace:
+          root: ~/project
+          paths:
+            - .
```

Comment:
This may need `workflows` block to run properly - you can see an example [here](https://circleci.com/docs/sample-config/?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-maxConv-auth-nb&utm_term=g_-_c__dsa_&utm_content=&gclid=Cj0KCQiAvqGcBhCJARIsAFQ5ke7GM3THzTRh2Xs5QuNAIIzYoxE2JO1sPNoqao2uln5yoYbxa2Xn0woaAjuAEALw_wcB#simple-configuration-examples) on line 25.

The `jobs` block looks great, good work!